src/main/webapp/app/entities/course/course.module.ts

```diff
@@ -33,6 +34,7 @@ const ENTITY_STATES = [...courseRoute];
         ArtemisColorSelectorModule,
         ImageCropperModule,
         MomentModule,
+        IntellijModule,
```

Comment:
please remove

Reply:
I checked and I can't remove this, since it provides the components with the `ide-filter.directive`, which is necessary for displaying/not displaying certain elements depending on the user agent

Reply:
ok